src/sap/fhir/model/r4/lib/FHIROperationOutcome.js

```diff
@@ -0,0 +1,118 @@
+/*!
+ * ${copyright}
+ */
+
+// Provides class sap.fhir.model.r4.lib.FHIROperationOutcome
+sap.ui.define([], function () {
+	"use strict";
+
+	/**
+	 * Constructor for a new FHIROperationOutcome.
+	 * @see [FHIR Specification]{@link https://www.hl7.org/fhir/operationoutcome.html}
+	 *
+	 * @param {object} oResource The FHIR resource
+	 * @constructs {sap.fhir.model.r4.lib.FHIROperationOutcome} Provides the implementation of a FHIR OperationOutcome
+	 * @alias sap.fhir.model.r4.lib.FHIROperationOutcome
+	 * @author SAP SE
+	 * @public
+	 * @since 2.0.0
+	 * @version ${version}
+	 */
+	var FHIROperationOutcome = function (oResource) {
+		this._sResourceType = oResource.resourceType;
+		this._aIssue = oResource.issue ? oResource.issue : [];
+	};
+
+	/**
+	* Get All Issues
+	*
+	* @returns {array} aIssues All the issues in the given instance
```

Comment:
using the jsdoc `@returns` no parameter names has to be specified. only for `@param` tags.